scripts/show_parse.py

```diff
@@ -56,6 +70,8 @@ def print_parse(source: str, verbose: bool = False) -> None:
 
 def main() -> None:
     args = parser.parse_args()
+    if args.diff and not args.grammar_file:
+        parser.error("--diff requires --grammar-file")
```

Comment:
```suggestion
        parser.error("--diff/-d requires --grammar-file/-g")
```

Reply:
Good idea, did this as a separate commit (no PR) since I already merged. :-)